### PR TITLE
Closes #8: automatic release on tag creation that matches the semver regex

### LINKED ISSUE #8: Automatic release
tgz for linux and mac and zip for windows, also we can use zip for all OS.

### PR DESCRIPTION
**Pull request by fllaca**
Closes #8 

Still WIP, asking for feedback

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -47,5 +85,13 @@ workflows:
...
+                - /^v[0-9]+(\.[0-9]+)*.+$/
+      - publish:
+          requires:
+            - build
+          filters:
+            branches:
+              ignore: /.*/
+            tags:
+              only:
+                - /^v[0-9]+(\.[0-9]+)*.+$/
```
Comment (figuerascarlos) [inline .circleci/config.yml]: not a regex expert. with automatic releases, I think we need to be more strict in the tags naming. for example
```                
- /^v[0-9]\d*\.[0-9]\d*\.[0-9]\d*$/
- /^v[0-9]\d*\.[0-9]\d*\.[0-9]\d*-rc[0-9]\d*$/
```
with these we force v0.0.0 and v.0.0.0-rc0 tags naming. we can merge in one expression. 

WDYT ??

**Comment by figuerascarlos** [comment]
with this PR, we can close #46

File: .circleci/config.yml (hunk 1 above)
Comment (fllaca) [reply .circleci/config.yml]: as rebased with master, this change is already done, thanks!

**Review by figuerascarlos — APPROVED** [review]

**Comment by figuerascarlos** [comment]
LGTM

**Pull request merged**